Xamarin.RangeSlider.Droid/RangeSliderControl.cs

```diff
@@ -759,8 +759,12 @@ protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
         /// </summary>
         protected override void OnDraw(Canvas canvas)
         {
-            base.OnDraw(canvas);
-
+            base.OnDraw(canvas);
+            if (ThumbImage != null) {

```

Comment:
This code should only sit in `UpdateThumbImages`.

Reply:
the only idea i have is to make an new property "customThumbImage" and don't use ThumbImage in the PlattformEffect

        private Bitmap _customThumbImage;
        public Bitmap CustomThumbImage 
        {
            get { return _customThumbImage; }
            set {
                _customThumbImage = value;
                UpdateThumbImages();
            }
        }

and in UpdateThumbImages()

            ThumbImage = DefaultThumbImage ?? (CustomThumbImage ?? BitmapUtil.DrawableToBitmap(GetThumbDrawable()));

thx for feedback